src/activators/reversey.js

```diff
@@ -0,0 +1,20 @@
+function ReverseY(el, button, onActivate) {
```

Comment:
Interesting use of an activator.. Not quite changing how the action activates but I guess it makes sense. It's definitely good to experiment with this so we can flesh out what kinds of features we want in AFIM